targets/TARGET_NUVOTON/TARGET_NUC472/crypto/crypto-misc.c

```diff
@@ -216,7 +216,11 @@ static bool crypto_submodule_acquire(uint16_t *submodule_avail)
 static void crypto_submodule_release(uint16_t *submodule_avail)
 {
     uint16_t expectedCurrentValue = 0;
```

Comment:
Isn't the expected value 1 here?

Reply:
It's a backwards "available" flag. So 0 when in use, and write 1 to release.